test/Main.spec.js

```diff
@@ -85,6 +85,17 @@ describe('The readout', function() {
             expect(btn).to.be.null;
         });
 
-    });
+        it("should change width of textarea", async function() {
+            const issueStyle = await page.evaluate(textarea => {
+                const sel = document.querySelector(textarea);
+                return JSON.parse(JSON.stringify(getComputedStyle(sel)));
+            }, SEL_TEXTAREA);
+
+            expect(issueStyle.width).to.eql('425px');
```

Comment:
Can you think of a better way to test that the width changed?

Reply:
Hi Janet, 
I rewrote how we are testing this by employing a helper function that returns the width of a selector. I also use a global variable that stores the original textarea width, which is then 'expected' to be greater than the new width found within this test block. This is the easiest method I could think of for testing that the width changed, but please let me what you think. Thanks!